tools/rebuild-elasticsearch-aip-index-from-files

```diff
@@ -146,96 +136,140 @@ def processAIPThenDeleteMETSFile(path, temp_dir, delete_existing_data = False):
         if aip_type == "Archival Information Collection":
             aips_in_aic = get_aips_in_aic(root, path, temp_dir)
 
-    elasticSearchFunctions.connect_and_index_files('aips', 'aipfile', aip_uuid, temp_dir)
-    elasticSearchFunctions.connect_and_index_aip(aip_uuid, aip_name, path, path_to_mets, aips_in_aic=aips_in_aic)
-    os.remove(path_to_mets)
+    try:
+        elasticSearchFunctions.index_aip(
+            client=es_client,
+            uuid=aip_uuid,
+            name=aip_name,
+            filePath=path,
```

Comment:
This should be updated to ask the SS for the path so it works when the AIPs aren't stored on the same machine (eg SS & AM on different VMs)